greenpithumb/db_store.py

```diff
@@ -2,6 +2,18 @@
 from dateutil import parser
 
 
+def _serialize_timestamp(timestamp):
+    """Converts a timestamp to a string.
+
+    Args:
+        timestamp: Timestamp in datetime.datetime format.
```

Comment:
Suggest just saying "Timestamp as a datetime object." The "format" makes it sound like it's already a string.